src/services/db-manager.ts

```diff
@@ -22,9 +20,13 @@ export async function migrate(conn: Connection) {
     await iasqlPlatform.migrations.afterInstall(qr);
   }
   await qr.query(`INSERT INTO iasql_module VALUES ('iasql_platform@${version}')`);
-  await iasqlFunctions.migrations.install(qr);
-  if (iasqlFunctions.migrations.afterInstall) {
-    await iasqlFunctions.migrations.afterInstall(qr);
+  await ModuleSet?.IasqlFunctions?.migrations?.install(qr);
```

Comment:
Looking forward to dropping a lot of weirdness in this file once `v0.0.16` is gone.

Reply:
Agree!